libs/community/langchain_google_community/vertex_ai_search.py

```diff
@@ -25,13 +24,29 @@
 from langchain_google_community._utils import get_client_info
 
 if TYPE_CHECKING:
-    from google.cloud.discoveryengine_v1beta import (  # type: ignore[import, attr-defined]
-        ConversationalSearchServiceClient,
-        SearchRequest,
-        SearchResult,
-        SearchServiceClient,
+    from google.cloud.discoveryengine_v1 import (  # noqa: I001
+        ConversationalSearchServiceClient as StableConversationalSearchServiceClient,
+        SearchRequest as StableSearchRequest,
```

Comment:
nits: let's call it `V1SearchRequest` and `V1BetaSearchRequest` for a better readability, please

or `SearchRequest` vs `BetaSearchRequest` maybe, but let's definitely drop `Stable`